### PR TITLE
actively supported versions of Node.js: 6, 8, and 9

### PR DESCRIPTION
**Pull request by selfup**
References #473

Update Travis Node Versions to Supported Versions 🎉

### CODE REVIEW

**Comment by selfup** [comment]
Looks like we are good to go!

screen shot 2017-12-04 at 7 08 45 am

**Comment by JorgeBucaran** [comment]
Thanks @selfup! :9

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
+  - "6"
   - "8"
-  - "7"
+  - "9"
```
Comment (frenzzy) [inline .travis.yml]: maybe `"stable"` instead of `"9"`? ..and only LTS versions define explicitly

Comment (selfup) [reply .travis.yml]: Good question. @JorgeBucaran what do you think?

Comment (JorgeBucaran) [reply .travis.yml]: What do you folks make out of this?

screen shot 2017-12-04 at 23 49 27

Comment (JorgeBucaran) [reply .travis.yml]: https://github.com/travis-ci/docs-travis-ci-com/issues/362

Comment (selfup) [reply .travis.yml]: lots of options 😂

**Pull request merged**